modules/migration-mtc-release-notes-1-7-13.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-13_{context}"]
+= {mtc-full} 1.7.13 release notes
+
+[id="resolved-issues-1-7-13_{context}"]
+== Resolved issues
```

Comment:
There's no preview link for me to check and I'm not sure where to find the migration toolkit stuff since I'm normally in OCP docs, and searching there doesn't seem to return the preview for this...so I'm unsure of the typical naming conventions here, but usually I see Bug Fixes instead of Resolved issues. I know its basically empty, but just wondering about the naming.

Reply:
Ah ok, I see you use Resolved issues throughout this release note. I guess we all do different things in these asynch release notes

Reply:
@skrthomas - yes. :(
i know... i inherited `Resolved issues`.